src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -491,27 +491,43 @@ internal Uri CreateIssuePreview(string projectName, string teamName, IReadOnlyDi
         }
 
         /// <summary>
-        /// Retains characters within ascii range below 127 and returns
-        /// an escaped representation of the string
+        /// Substitutions for characters when creating the bug URL
+        /// </summary>
+        internal static IReadOnlyDictionary<char, char> CharacterSubstitutions = new Dictionary<char, char>()
+        {
+            { '�', '-' }, // middle dot (183) to dash (45)
```

Comment:
Let's be very explicit how the unicode values are represented in code (especially since this file doesn't have a UTF encoding header)
```suggestion
            { '\B7', '-' }, // middle dot (183) to dash (45)
```